specificatie/verblijfplaats.yaml

```diff
@@ -193,14 +186,12 @@ components:
           $ref: '#/components/schemas/Locatiebeschrijving'
         land:
           $ref: 'common.yaml#/components/schemas/Waardetabel'
-        gemeenteVanInschrijving:
-          $ref: 'common.yaml#/components/schemas/Waardetabel'
         adresregel1:
```

Comment:
moet dit niet ook regel1 heten? We gebruiken nu regel1 als buitenlands adres en adresregel1 als samengesteld adres. lijkt me verwarrend om in de gba versie zelfde veld andere betekenis te geven.

Reply:
Ben ik wel met je eens. @ Melvin, jij ook ?

Reply:
in de verwachting dat Melvin het hier ook mee eens is alvast aangepast.